mod/dfrn_request.php

```diff
@@ -449,9 +449,13 @@ function dfrn_request_post(&$a) {
 			$data = probe_url($url);
 			$network = $data["network"];
 
+			logger('dfrn_request: url=' . $url . ',network=' . $network . ',hcard=' . $hcard . ' - BEFORE!', LOGGER_DEBUG);
```

Comment:
Normally I would keep my debug lines around like this:
`//* NOISY-DEBUG: */ doSomeLogging('bla foo bar');`
This allows me to easily comment them in by just removing one slash. If I don't want them anymore, I just re-add the slash and the line is no longer executed.